lib/utils/stream.js

```diff
@@ -113,4 +116,8 @@ Stream.prototype.flush = function() {
   this.trigger('done');
 };
 
+Stream.prototype.error = function(error) {
```

Comment:
makes sense I will remove the error function from stream and change it for trigger('error', ...);